package.json

```diff
@@ -47,15 +47,18 @@
     "dependents-filter:semistandard": "npm run --silent dependents-filter-one -- -i dependents-data/source/semistandard.ndjson -o dependents-data/semistandard-filtered.ndjson --min-downloads=1000 --target-version='>=16.0.0' --exclude $(jq -r 'keys | join(\" --exclude \")' dependents-data/known-failures/semistandard.json)",
     "dependents-filter:standard": "npm run --silent dependents-filter-one -- -i dependents-data/source/standard.ndjson -o dependents-data/standard-filtered.ndjson --min-downloads=75000 --target-version='>=17.0.0' --exclude $(jq -r 'keys | join(\" --exclude \")' dependents-data/known-failures/standard.json)",
     "dependents-filter:ts-standard": "npm run --silent dependents-filter-one -- -i dependents-data/source/ts-standard.ndjson -o dependents-data/ts-standard-filtered.ndjson --min-downloads=1000 --exclude $(jq -r 'keys | join(\" --exclude \")' dependents-data/known-failures/ts-standard.json)",
-    "dependents-filter": "run-p dependents-filter:*",
-    "dependents-refresh-one": "cd dependents-data/source/ && list-dependents refresh --field engines -n",
+    "dependents-filter-simplify": "for PROJECT in $(jq -r 'join(\" \")' dependents-data/projects.json); do jq -s '[.[].repositoryUrl | sub(\"https://github.com/\";\"\";\"i\")] | unique | sort' \"dependents-data/$PROJECT-filtered.ndjson\" > \"dependents-data/$PROJECT.json\"; done",
```

Comment:
Used to be done at runtime in [.github/workflows/compatibility.yml](https://github.com/neostandard/neostandard/pull/121/files#diff-0ff6bcc45c7c09bfcb5419c83559f190add597ec3b82ad238ddb1d6e43e9517e) – condensed the filtered dependency lists into an alphabetically sorted list of unique GitHub repository names, which is a much more stable format to do diffs on